packages/contracts-bedrock/src/L2/WETH.sol

```diff
@@ -14,8 +14,8 @@ import { IL1Block } from "src/L2/interfaces/IL1Block.sol";
 /// @title WETH contract that reads the name and symbol from the L1Block contract.
 ///        Allows for nice rendering of token names for chains using custom gas token.
 contract WETH is WETH98, ISemver {
-    /// @custom:semver 1.1.0-beta.2
-    string public constant version = "1.1.0-beta.2";
```

Comment:
why are we changing the version of this if we are not changing anything?

Reply:
We're modifying the contract it inherits, that produces changes on the `WETH`. AFAIK if I don't update the semver and there are changes on the contract, the `just pre-pr` will fail when it checks that no changes were found on contracts without any version modification

Reply:
perfect, missed this inherits from `WETH98` thought it was the other way around